intellij/dependencies.json

```
@@ -0,0 +1,26 @@
+{
+  "[2025.1,)": [
+    {
+      "tool": "java",
+      "versionRange": "[21,)"
+    }
+  ],
+  "[2024.1,2025.1)": [
+    {
+      "tool": "java",
+      "versionRange": "[21,)"
+    }
+  ],
+  "[2022.2,2024.2)": [
+    {
+      "tool": "java",
+      "versionRange": "[17,)"
+    }
+  ],
+  "[2020.3,2022.1)": [
```

Versions like `2022.1.4` currently have no dependency at all
```suggestion
  "[2020.3,2022.2)": [
```